README.md

```diff
@@ -235,7 +235,7 @@
 ### Geometry ![problems-solved](https://img.shields.io/badge/Solved-0/30-00ffff.svg)
 | Name | Difficulty | Solution |
 |------|:----------:|:--------:|
-| [Points On A Line](https://www.hackerrank.com/challenges/points-on-a-line) | Easy | |
+| [Points On A Line](https://www.hackerrank.com/challenges/points-on-a-line) | Easy |![cpp](https://img.icons8.com/color/35/000000/c-plus-plus-logo.png) |
```

Comment:
Thanks for adding this solution, and please continue contributing 👍 - just a small thing we want that people are able to click on the link and then see the solution, so the image that we have added here should be a link. To make it inti a link do the following:

`[![img-title](image-link)](cpp/points_on_a_line.cpp)`

so it will be 

```md
[![cpp](https://img.icons8.com/color/35/000000/c-plus-plus-logo.png)](cpp/points_on_a_line.cpp)
```

also please rename the folder `C++` to --> `cpp`. Thank you for contributing 😄